### ISSUE TITLE
Hard to identify a plot in editor tab / title is not descriptive enough

### ISSUE THREAD

**Issue by timtmok** [posit-dev/positron#5895]
#### Positron and OS details:
2025.01

## Describe the issue:
The editor tab title for plots uses the plot id, which is a GUID. It's long and hard to use to distinguish plots. There needs to be something more obvious and easier to remember which plot is which without having to switch to the plot to see what it is.

## Steps to reproduce the issue:

1. Create multiple plots
2. Open them in editors

## Expected or desired behavior:
The titles for the editor tabs should be easily identifiable as to which plot it will be.

**Comment by juliasilge** [comment]
Is this a duplicate of #5872?

**Comment by timtmok** [comment]
> Is this a duplicate of [#5872](https://github.com/posit-dev/positron/issues/5872)?

That was the fastest reaction to a duplicate that I've ever seen!
